pubspec.yaml

```diff
@@ -14,6 +14,8 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.1.6
+  firebase_admob: 0.9.0+7
```

Comment:
I don't think we want to depend on `firebase_admob` If you'd like to add targeting info it should be implemented in this code base.

Reply:
If you want to add `MobileAdTargetingInfo` to the dart side here I can help with the Swift side.